lib/browser/api/web-contents.ts

```diff
@@ -360,14 +357,14 @@ WebContents.prototype.print = function (options = {}, callback) {
           throw new Error('height and width properties are required for pageSize');
         }
         // Dimensions in Microns - 1 meter = 10^6 microns
-        options.mediaSize = {
+        (options as any).mediaSize = {
```

Comment:
These `any` casts shouldn't be needed if the original `options` object is typed correctly